File: hw/fuentes/U3/HW22STRATEGY/SortAppPY/ec/edu/espe/clientApp/sort_app.py
from model.sorting_context import SortingContext

def main():
    data = [3, 6, 4, 6, 7, 8, 5, 6, 7, 5, 3, 3]
    context = SortingContext()
    sorted_data = context.sort(data)
    print("Sorted array:", sorted_data)

if __name__ == "__main__":
    main()

File: hw/chicaiza/u3/HW22Strategy/sortapp/src/main/java/ec/edu/espe/sort/view/SortApp.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package ec.edu.espe.sort.view;

import ec.edu.espe.sort.model.SortingContext;
import java.util.Arrays;

/**
 *
 * @author Chicaiza Michael
 */
public class SortApp {

    public static void main(String[] args) {
        int data[] = {3,6,4,6,7,8,5,6,7,5,3,3};
        SortingContext sc = new SortingContext();
        int sortedList[] = sc.sort(data);
        
        System.out.println("Sorted array: " + Arrays.toString(sortedList));
    }
}
